### ISSUE TITLE
[REVIEW]: RHEOS - A Julia package for Rheology Data Analysis.

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#1700]
**Submitting author:** @akabla (<a href="http://orcid.org/0000-0002-0280-3531">Alexandre Kabla</a>)
**Repository:** <a href="https://github.com/JuliaRheology/RHEOS.jl" target ="_blank">https://github.com/JuliaRheology/RHEOS.jl</a>
**Version:** v0.9.1
**Editor:** @jedbrown
**Reviewer:** @adambeall, @gbruer15, @HaoZeke
**Archive:** <a href="https://doi.org/10.5281/zenodo.3458310" target="_blank">10.5281/zenodo.3458310</a>

## Status

[![status](https://joss.theoj.org/papers/553250d815e1990db1b89c742854c71a/status.svg)](https://joss.theoj.org/papers/553250d815e1990db1b89c742854c71a)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/553250d815e1990db1b89c742854c71a"><img src="https://joss.theoj.org/papers/553250d815e1990db1b89c742854c71a/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/553250d815e1990db1b89c742854c71a/status.svg)](https://joss.theoj.org/papers/553250d815e1990db1b89c742854c71a)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/JuliaRheology/RHEOS.jl" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@adambeall & @gbruer15 & @HaoZeke, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @jedbrown know.

✨ **Please try and complete your review in the next two weeks** ✨ 

### Conflict of interest

- [x] As the reviewer I confirm that I have read the [JOSS conflict of interest policy](https://github.com/openjournals/joss/blob/master/COI.md) and that there are no conflicts of interest for me to review this work.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/JuliaRheology/RHEOS.jl">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@akabla) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] As the reviewer I confirm that I have read the [JOSS conflict of interest policy](https://github.com/openjournals/joss/blob/master/COI.md) and that there are no conflicts of interest for me to review this work.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/JuliaRheology/RHEOS.jl">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@akabla) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] As the reviewer I confirm that I have read the [JOSS conflict of interest policy](https://github.com/openjournals/joss/blob/master/COI.md) and that there are no conflicts of interest for me to review this work.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/JuliaRheology/RHEOS.jl">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@akabla) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @adambeall, @gbruer15, @HaoZeke it looks like you're currently assigned to review this paper :tada:.

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01700/joss.01700/10.21105.joss.01700.pdf)

**Comment by jedbrown** [comment]
@adambeall, @gbruer15, @HaoZeke :wave: Welcome and thanks for agreeing to review! The comments from @whedon above outline the review process, which takes place in this thread (possibly with issues filed in the [RHEOS.jl repository](https://github.com/JuliaRheology/RHEOS.jl)). I'll be watching this thread if you have any questions.

**Comment by akabla** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01700/joss.01700/10.21105.joss.01700.pdf)

**Comment by akabla** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01700/joss.01700/10.21105.joss.01700.pdf)

**Comment by adambeall** [comment]
Thank you for the opportunity to review. I have enjoyed trying out RHEOS and think that it is accessible, the examples hosted in the repository nicely demonstrate its functionality and the documentation is thorough. RHEOS should be useful for researchers from a range of disciplines teaching and forward modelling a variety of linear viscoelastic rheologies, as well as fitting experimental data (the Bonfanti et al., 2019 demonstration is impressive). 

Thanks to the authors for making the minor recommended changes. While [issue 43](https://github.com/JuliaRheology/RHEOS.jl/issues/43) is still open, I will be unavailable for the next week and don’t want to hold up the review, so I have completed the checklist and recommend acceptance after the issue is closed.

**Comment by akabla** [comment]
Thank you so much Adam for your support and all your input and comments throughout. We will address issue 43 very soon - you made a good point indeed, and it should be straight-forward to address.

**Comment by HaoZeke** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Attempting to check references...
```

**Comment by whedon** [comment]
```Reference check summary:

OK DOIs

- 10.1137/141000671 is OK
- 10.1101/543330 is OK
- 10.1016/j.mechmat.2013.09.017 is OK
- 10.1101/565614 is OK
- 10.1017/CBO9780511626722 is OK
- 10.1122/1.5025622 is OK
- 10.1122/1.5018715 is OK

MISSING DOIs

- https://doi.org/10.1115/1.3424077 may be missing for title: Creep and Relaxation of Nonlinear Viscoelastic Materials: With an Introduction to Linear Viscoelasticity

INVALID DOIs

- None
```

**Comment by gbruer15** [comment]
I've finished my review and recommend accepting this submission, pending issue [#43](https://github.com/JuliaRheology/RHEOS.jl/issues/43). Other than that issue, the software's documentation is in a good, user-friendly state, and I believe this package is a very useful contribution to rheology.

**Comment by HaoZeke** [comment]
I have completed my review @jedbrown, and as soon as https://github.com/JuliaRheology/RHEOS.jl/issues/43 is closed satisfactorily, I recommend this for publication. RHEOS is very well documented and easily extensible, it's really well structured and addresses an existing gap in the rheology community. Glad to have been introduced to it, @akabla.

**Comment by moustachio-belvedere** [comment]
Many thanks from me too @adambeall @gbruer15 @HaoZeke for all your valuable feedback and being so efficient in working through everything. We are very grateful.

**Comment by jedbrown** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01700/joss.01700/10.21105.joss.01700.pdf)

**Comment by moustachio-belvedere** [comment]
Thank you also very much @jedbrown for editing. Hopefully you are satisfied with everything so far, and please do let us know any next steps that we might need to take.

**Comment by moustachio-belvedere** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01700/joss.01700/10.21105.joss.01700.pdf)

**Comment by jedbrown** [comment]
Thanks for your patience.
* Would it make sense to name the authors of Bob-rheology? https://sourceforge.net/p/bob-rheology/wiki/Home/  There are three authors listed in the copyright statements inside the repository, with version 2.5 release in 2012. Perhaps we can name the three authors and use `year = 2012`. I've suggested that in my PR https://github.com/JuliaRheology/RHEOS.jl/pull/49
* Where does the year 1989 come from for [this book](https://store.doverpublications.com/0486660168.html )? It looks to be a [1976 book](https://watermark.silverchair.com/364_1.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAmwwggJoBgkqhkiG9w0BBwagggJZMIICVQIBADCCAk4GCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMKW5euSTlIsgNAvRZAgEQgIICH-dgrsCFgUgCwFr6n9BevlIHKG1UKGyLXfwsoVjfONgNmdfmfQjUxLTpUfJQrwMN0eSaq3Xr_7iliGvzlQnuBjmZf2K5RV8VWwtIa6CQEn5irVdUvcsXEtoI55pmZKEDZ5FJacZhTml7svEe25eRJPDhcB-D0c27WOMIQPwzmXpbMjhEo1M4bAZNNQwAz580I6Y9XbxehGmoHk9J0Z1TE_tVuObGLTnneLaOmHWRHUzdzUz8NK7-9B0-n_S40mgZHY9OsF3tFMksBIFhjsnMY_3-XrO9TewMVxiWyCOb3FLc390XYhVEJwQNLa9fBJsOdQe0P0DHSxU9NkHnR3RwSBKwySBX6iGLrIDlZgCsG2MgnKhDcvhsE0yqT9TVzmsbhMZhn1im9Mwi2hb31k3jXjSnzIW1APlQCZF_Oi5_TfWs5bJueJSvY3zZPH3AcALGfNns3oknHzgXrNK1eu5ndqKV1MeVWe3jz9n2OhVD7lntDd1ff5HMyEOQNMAMkDOOaJsujAkmgtqJgc9_AOiJV7pnBGo30FMkfMbCuTFr6HCPvbVh6JPEakLdjbsFtTY9K_ClIuAhuRVlHD01N7_-64_SWv-t9LW6nzDQGwjQae-Fm3CXnABA_QsfTwSyt23wclrQySSCNApmSkm_A6b6kgijwd6S4rth3wSW8ZYETh3m1NDP0tDlGNzofdZj_GEbAMqHIMNyE7UNDEZrSn9jFg) reprinted in 2011. If the year needs to change, the sentence in which it's mentioned will have to be updated as well.
* Perhaps you could rephrase that sentence in which you name many textbooks to just say "over the past N decades" because the APA-style citations include the year anyway.

**Comment by moustachio-belvedere** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01700/joss.01700/10.21105.joss.01700.pdf)

**Comment by moustachio-belvedere** [comment]
@jedbrown Many thanks for your feedback. On your first point I agree completely, thanks a lot for the pull request.

On the third point I also agree completely, I hadn't noticed the referencing style and have updated the wording in line with your suggestion, commit https://github.com/JuliaRheology/RHEOS.jl/commit/2186c1d9efabf3f69e009780cb051c061ca65681

Regarding the book reference, it does indeed come from the Dover publications version. However, I believe their first reprint was in 1989. The front matter of the version I have states the following:

_Copyright @ 1976 by North-Holland Publishing Company.
All rights reserved.

This Dover edition, first published in 1989, is an unabridged, corrected
republication of the work originally published in 1976 by North-Holland
Publishing Company, Amsterdam, as Volume 18 in the “North-Holland
Series in Applied Mathematics and Mechanics.” A new section of additional
literature relating to Chapter 2 has been added in this edition at the end of the
bibliography._

I would also note that the 1989 version is sometimes cited, e.g. in the popular MIT lecture notes:

http://web.mit.edu/course/3/3.11/www/modules/visco.pdf

As I don't know exactly how much was abridged in the original 1976 version I might be hesitant to cite that edition. But if there are referencing precedents that the first edition should be cited then I am more than happy to oblige. I also very happy to check with our department librarian if there are precedents that they are aware of regarding this.

Many thanks again.

**Comment by moustachio-belvedere** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01700/joss.01700/10.21105.joss.01700.pdf)

**Comment by moustachio-belvedere** [comment]
Another update, for now  I tried to add an `origdate` field with `1976` embedded but it does not seem to be recognised by the compiler backend pipeline, perhaps there is another field which could be used. The addition was based on this discussion:

https://tex.stackexchange.com/questions/139805/how-to-differentiate-between-translated-and-reprinted-work-with-apa-style

Many thanks

**Comment by moustachio-belvedere** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01700/joss.01700/10.21105.joss.01700.pdf)

**Comment by moustachio-belvedere** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01700/joss.01700/10.21105.joss.01700.pdf)

**Comment by jedbrown** [comment]
I tried following [this example](https://github.com/jgm/pandoc-citeproc/blob/master/tests/biblatex-examples.bib#L809-L836) from `pandoc-citeproc` (the software pipeline we use), but those fields apparently are not supported, so I'm fine with this as is.

Please tag a release ([annotated tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging#_annotated_tags) preferred) and archive your software on Zenodo or similar, then report the DOI back here.

**Comment by akabla** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01700/joss.01700/10.21105.joss.01700.pdf)

**Comment by akabla** [comment]
Dear Jed @jedbrown,

here is the submission doi:
10.5281/zenodo.3458310

Many thanks for your help editing this paper!

**Comment by jedbrown** [comment]
@akabla Please edit the author info on Zenodo to match the paper. (Kyle and myself should not be authors.) Looks good otherwise and we're ready to accept.

**Comment by jedbrown** [comment]
@whedon set v0.9.1 as version

**Comment by whedon** [comment]
OK. v0.9.1 is the version.

**Comment by akabla** [comment]
Thanks @jedbrown for spotting this. It should be fixed now.
[https://doi.org/10.5281/zenodo.3458310](https://doi.org/10.5281/zenodo.3458310)

**Comment by jedbrown** [comment]
@whedon set 10.5281/zenodo.3458310 as archive

**Comment by jedbrown** [comment]
@whedon accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/976

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/976, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by jedbrown** [comment]
@openjournals/joss-eics This paper is now accepted; over to you.

**Comment by danielskatz** [comment]
Thanks @jedbrown - will process shortly

(and a minor point on the terminology - it's not accepted yet, but will be once the final processing is done)

**Comment by danielskatz** [comment]
Thanks to @adambeall & @gbruer15 & @HaoZekefor reviewing and to @jedbrown for editing!

**Comment by danielskatz** [comment]
@whedon accept deposit=true

**Comment by whedon** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by whedon** [comment]
🐦🐦🐦 👉 [Tweet for this paper](https://twitter.com/JOSS_TheOJ/status/1176499941612675072) 👈 🐦🐦🐦

**Comment by whedon** [comment]
🚨🚨🚨 **THIS IS NOT A DRILL, YOU HAVE JUST ACCEPTED A PAPER INTO JOSS!** 🚨🚨🚨

 Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/openjournals/joss-papers/pull/977
1. Wait a couple of minutes to verify that the paper DOI resolves [https://doi.org/10.21105/joss.01700](https://doi.org/10.21105/joss.01700)
2. If everything looks good, then close this review issue.
3. Party like you just published a paper! 🎉🌈🦄💃👻🤘

 Any issues? notify your editorial technical team...

**Comment by whedon** [comment]
:tada::tada::tada: Congratulations on your paper acceptance! :tada::tada::tada:

If you would like to include a link to your paper from your README use the following code snippets:

```
Markdown:
[![DOI](https://joss.theoj.org/papers/10.21105/joss.01700/status.svg)](https://doi.org/10.21105/joss.01700)

HTML:
<a style="border-width:0" href="https://doi.org/10.21105/joss.01700">
  <img src="https://joss.theoj.org/papers/10.21105/joss.01700/status.svg" alt="DOI badge" >
</a>

reStructuredText:
.. image:: https://joss.theoj.org/papers/10.21105/joss.01700/status.svg
   :target: https://doi.org/10.21105/joss.01700
```
This is how it will look in your documentation:

[![DOI](https://joss.theoj.org/papers/10.21105/joss.01700/status.svg)](https://doi.org/10.21105/joss.01700)

**We need your help!**

Journal of Open Source Software is a community-run journal and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to review for us sometime in the future. You can add your name to the reviewer list here: https://joss.theoj.org/reviewer-signup.html
- Making a small donation to support our running costs here: https://numfocus.salsalabs.org/donate-to-joss

**Comment by moustachio-belvedere** [comment]
Thanks very much for editing @jedbrown .

And thankyou @danielskatz for finalizing the process.

The entire review process has been very valuable to us.
